src/modules/shared/pages/account/manage-details/manage-details-edit.component.ts

```diff
@@ -108,7 +108,7 @@ export class PageAccountManageDetailsEditComponent extends CoreComponent impleme
 
     const body: UpdateUserInfoDTO = {
       displayName: wizardData.displayName,
-      ...(wizardData.mobilePhone ? { mobilePhone: wizardData.mobilePhone } : {}),
+      ...(wizardData.mobilePhone ? { mobilePhone: wizardData.mobilePhone } : { mobilePhone: null }),
```

Comment:
As the key is always defined, the code could be simpler by:
mobilePhone: wizardData.mobilePhone ?? null

Thoughts?

Reply:
Won't work because the value can be either null or empty string